recipes/jfalcou-eve/all/conanfile.py

```diff
@@ -28,18 +26,28 @@ def _min_cppstd(self):
 
     @property
     def _compilers_minimum_version(self):
-        return {
-            "gcc": "10.2",
-            "Visual Studio": "16.9",
-            "clang": "12",
-            "apple-clang": "13",
-        }
+        if tools.Version(self.version) >= "2022.03.0":
```

Comment:
```suggestion
        if not self.version.startswith("cci.") and  tools.Version(self.version) >= "2022.03.0":
```

Reply:
oh! forgot I left that here. Thanks for the fix and sorry for forgetting XD